### PR TITLE
Changed all instances of Pano to Sphere

### PR DESCRIPTION
**Pull request by Dumamax**
There are a few minor fixes to the code as well, but the "major" changes will come in another commit

### CODE REVIEW

File: 11-look-at/demo.js
Comment (bobsbeenjamin) [inline 11-look-at/demo.js:1]: I'm trusting that your number changes here look in the correct directions. The numbers are believable, at least.

File: 27-set-pano-marker-distance/demo.details [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
-name: AerialSphere Set Pano Marker Distance
-description: Sets the max distance at which to render panorama markers
+name: AerialSphere Set Sphere Marker Distance
+description: Sets the max distance at which to render sphere (panorama) markers
```
Comment (bobsbeenjamin) [inline 27-set-pano-marker-distance/demo.details:3]: We can probably eliminate sphere, and just say `render markers`

File: 34-set-fov-range/demo.js [hunk 2]
```diff
@@ -34,11 +34,11 @@ const aerialSphere = new AerialSphere(
 );
 
 const setFovMin = (e) => {
-    foxMax = document.getElementById('fovMax');
+    fovMax = document.getElementById('fovMax');
```
Comment (bobsbeenjamin) [inline 34-set-fov-range/demo.js:37]: It looks like you were trying to line up function names (`setFovMin` and `setFovMax`) with functionality. But it still reads as reversed (ie, min acts on max, and vice versa). Maybe just change the function names themselves, and then make sure they get called correctly?

**Review by bobsbeenjamin — CHANGES_REQUESTED** [review]

File: 27-set-pano-marker-distance/demo.details (hunk 1 above)
Comment (Dumamax) [reply 27-set-pano-marker-distance/demo.details:3]: This is only for rendering sphere markers though, so I could see it being confusing to state "Sets the max distance at which to render markers"

File: 34-set-fov-range/demo.js (hunk 2 above)
Comment (Dumamax) [reply 34-set-fov-range/demo.js:37]: Well the only things I changed were the typos: foxMin & foxMax -> fovMin & fovMax, so I'm not entirely sure what you're meaning by this, but I will go back in and see if I can change some things around to fix it

Comment (Dumamax) [reply 34-set-fov-range/demo.js:37]: I went back and looked through it again, and I'm not sure I see any issues with the code. I think it'll work as expected, but if I'm missing something, definitely let me know. @bobsbeenjamin

**Review by Dumamax — COMMENTED** [review]
Except for the "setFovMin/Max" issue, everything else seems good, and comments have been responded to

File: 27-set-pano-marker-distance/demo.details (hunk 1 above)
Comment (bobsbeenjamin) [reply 27-set-pano-marker-distance/demo.details:3]: Yes. You are right. I guess my real issue is with leaving `panorama` in the verbiage. I thing that threw me off.

File: 34-set-fov-range/demo.js (hunk 2 above)
Comment (bobsbeenjamin) [reply 34-set-fov-range/demo.js:37]: Fixing the typo is better than not fixing it, so I'll approve this either way.
But it should be noted that your fix exposed a pre-existing issue: We define a function called `setFovMin` (on line 36), then immediately start dealing with `fovMax` (not min) on the first statement in the function (line 37).

Comment (Dumamax) [reply 34-set-fov-range/demo.js:37]: Ahh, I see what you mean. I think the confusing part with this is that both setFovMin and setFovMax are doing essentially the same thing, they are both using setFovRange(fovMin, fovMax). Both are just setting the min AND the max. I'll switch this up so it just sets the min, and just sets the max for the appropriate functions

**Review by bobsbeenjamin — APPROVED** [review]
Pushing this through

**Pull request merged**